posts/qrack_joins_uf.html

```diff
@@ -0,0 +1,169 @@
+                                <p><a href="https://qrack.net">Given Qrack’s support for Wasm</a>, we’ve joked at the Unitary Fund that Qrack can theoretically run on literal <a href="https://deviceatlas.com/blog/which-devices-have-browsers">internet-of-things enabled smart refrigerators</a>, but <a href="https://github.com/unitaryfund/qrack/issues/">open an issue ticket on the repository</a> if we’re blowing smoke and you still need developer support for your smart fridge, seriously!</p>
+                                <p style="font-weight: bold"><strong>QFT Benchmark</strong></p>
+                                <p>A standard example of Qrack (and PyQrack) performance is the quantum Fourier transform (QFT). Comparative results from benchmarking are in the graph below and are <a href="https://metriq.info/Task/142">available on Metriq</a> (as single-valued exponential trendline slope comparisons). Historically, Qrack measures this benchmark with an initialization by random 3-parameter unitary single qubit gates across the register width for a fair comparison, because Qrack QFT performance on computational basis states is classically efficient because of the underlying Schmidt decomposition. That wouldn’t be representative of general demands for cases of the QFT. While Qrack would perform worse for a random fully entangled case, our choice of randomly initialized separable qubits generally become fully entangled by the end of the circuit, paying a representative performance penalty for potentially “composing” every initially separable qubit. Hence this has historically been deemed a “fair” benchmark by the Qrack team.<p>
+                                <p>PyQrack has a commanding performance lead on this benchmark compared to other popular quantum computer simulators like Qiskit Aer (GPU) and QCGPU, where these three candidates are GPU-based simulators. Qiskit Aer (CPU) and Cirq (CPU) have been included on the chart, run on the same stock gaming laptop as the three GPU simulators, but remember that these software benchmarks are most fairly compared on identical hardware, whereas it isn’t possible for a CPU model and a GPU model to be the exact same hardware instance, obviously. However, PyQrack’s “hybrid CPU/GPU” methods utilize CPU simulation at the low qubit width end of the trend, which should be a fair comparison against CPU simulators until the user-selectable GPU switching threshold, which is 9 qubits in this case.</p>
```

Comment:
I wonder if the image should go after this paragraph?

Reply:
@willzeng What do you think? I thought the same thing as Nate.